src/main/java/com/assertthat/selenium_shutterbug/core/Shutterbug.java

```diff
@@ -97,7 +97,7 @@ public static PageSnapshot shootPage(WebDriver driver, ScrollStrategy scroll, in
             case BOTH_DIRECTIONS:
                 pageScreenshot.setImage(browser.takeScreenshotEntirePage());
             case WITHOUT_STICKY_HEADER:
-                pageScreenshot.setImage(browser.takeScreenshotEntirePage(true));
+                pageScreenshot.setImage(browser.takeScreenshotEntirePageUsingChromeCommand());
```

Comment:
Shall we add broser.getName(); which will check which driver instance is currently used (instanceof presumably) and throw UnsupportedOperationException if current driver instance is other than Chrome

Reply:
Also, please add missing break to BOTH_DIRECTIONS case